internal/kubernetes/operator/resources/normalizer.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+func AtlasNameToKubernetesName() map[string]string {
+	return map[string]string{
+		" ": "-",
+		".": "dot",
+		"@": "at",
+		"(": "left-parenthesis",
+		")": "right-parenthesis",
+		"&": "and",
+		"+": "plus",
+		":": "colon",
+		",": "comma",
+		"'": "single-quote",
```

Comment:
@Sugar-pack  I think one more character is missing - the underscore: `_`

Reply:
Done